.azure-pipelines/test-docker-sonic-vs-template.yml

```diff
@@ -24,6 +24,10 @@ jobs:
   pool: sonic-common
 
   steps:
+  - script: |
+      sudo rm -rf $(ls -A1)
```

Comment:
>sudo rm -rf $(ls -A1) [](http://example.com/codeflow?start=6&length=21)

This combination has risk. For example there is a filename as 'a b', and `rm` will try `rm a b` which is two files.

And `ls` could output colors in ANSI sequence.

Reply:
@qiluo-msft , the command is running before any script start, so it is to make the whole workspace clean. It is expected, there are no files as the agent hosted by Microsoft.